_data-prepper/pipelines/configuration/processors/service-map-stateful.md

```diff
@@ -14,12 +14,17 @@ Uses OpenTelemetry data to create a distributed service map for visualization in
 
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
-window_duration | No | Integer | Represents the fixed time window in seconds to evaluate service-map relationships. Default is 180.
+window_duration | No | Integer | Represents the fixed time window in seconds to evaluate service-map relationships. Default value is 180.
 
 <!---## Configuration
 
-Content will be added to this section.
+Content will be added to this section.--->
 
 ## Metrics
 
-Content will be added to this section.--->
\ No newline at end of file
+In addition to the metrics from [AbstractProcessor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/processor/AbstractProcessor.java):
+
+<!--- This is incomplete. Add the following? "Apart from common metrics in the [Abstract processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/processor/AbstractProcessor.java)"--->
```

Comment:
@graytaylor0 This is incomplete. Add the following? "Apart from common metrics in the [Abstract processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/processor/AbstractProcessor.java)"

Reply:
Would be similar to the other mentions ending in "the service-map-stateful processor introduces the following custom metrics